app/js/controllers/base.js

```diff
@@ -0,0 +1,9 @@
+export default class BaseController {
```

Comment:
I've migrated only things we actually use to have less technical debt and bias as possible (so no teardown and "view.init()" method).